src/charts/c3/c3.mixin.js

```diff
@@ -0,0 +1,57 @@
+angular.module('patternfly.charts').factory('ChartsMixin', function() {
+    'use strict';
+
+    // Donut Chart Defaults
+    var getDefaultDonut = function(title) {
```

Comment:
Don't we usually put a space after the `function` keyword ala `function (title) {`